src/compose.py

```diff
@@ -9,8 +9,13 @@ def start(settings):
 
 
 def stop(settings):
-    command = "down" if settings["persist_data"] else "down --volumes"
```

Comment:
`down` brings down the network which we don't want - we just want to stop the images. If `persist_data` is true we can also `rm` the images, meaning when we run `system prune` the volumes will also be cleaned up.